docs/content/docs/dev/table/hive-compatibility/hive-dialect/create.md

```diff
@@ -27,7 +27,7 @@ With Hive dialect, the following CREATE statements are supported for now:
 - CREATE DATABASE
 - CREATE TABLE
 - CREATE VIEW
-- CREATE MARCO
+- CREATE MACRO
```

Comment:
looks good - could you correct the Chinese versions and the instances in drop.md as well please.